hooks/useAlternateObservationZones.ts

```diff
@@ -0,0 +1,123 @@
+import {UseQueryResult, useQuery} from '@tanstack/react-query';
+import {polygon} from '@turf/helpers';
+import axios, {AxiosError} from 'axios';
+import {safeFetch} from 'hooks/fetch';
+import {LoggerContext, LoggerProps} from 'loggerContext';
+import {useContext} from 'react';
+import {xml2json} from 'xml-js';
+interface KMLPoint {
+  coordinates: string;
+}
+
+interface KMLPlacemark {
```

Comment:
Please write a `zod` schema (or generate one from the interface here, or a known good KML library like [togeojson](https://www.npmjs.com/package/@tmcw/togeojson)) so we can parse and validate our incoming data.